src/did.ts

```diff
@@ -131,6 +139,23 @@ export class DID {
     return this._capability
   }
 
+  /**
+   * Get parent DID, parent DID is the capability issuer
+   */
+  get parent(): string {
+    if (!this._parentId) {
+      throw new Error('DID has no parent DID')
+    }
+    return this._parentId
+  }
```

Comment:
why is this not just a getter that might return undefined/null if `this._parentId` is not set?
Then we wouldn't need the `hasParent` getter either.

Also, if `this._capability` is set, the issuer of that should be returned right?

Reply:
yeah just pattern matching the other getters in here (did - id, authenticated, capability), but actually does seem better for types

and yes, parent set in constructor when given capability, and if given both they will match